### PR TITLE
Changing default file format for the snippets from TOML to YAML

### PR DESCRIPTION
**Pull request by tomMoulard**
### What does this PR do?

Changing default file format in the documentation to have example in YML format first.

### Motivation

Unified documentation

### More

- ~~[ ] Added/updated tests~~
- [X] Added/updated documentation

### Additional Notes

Also removed some trailing white space.

### CODE REVIEW

File: docs/content/routing/entrypoints.md [hunk 1]
```diff
@@ -355,33 +345,33 @@ Setting them has no effect for UDP entryPoints.
...
+
     _Optional, Default=180s_
-    
-    `idleTimeout` is the maximum duration an idle (keep-alive) connection will remain idle before closing itself.  
-    
-    If zero, no timeout exists.  
+
+    `idleTimeout` is the maximum duration an idle (keep-alive) connection will remain idle before closing itself.
+
+    If zero, no timeout exists.
```
Comment (rtribotte) [inline docs/content/routing/entrypoints.md:371]:
```suggestion
    If zero, no timeout exists. 
```

File: docs/content/reference/dynamic-configuration/kubernetes-gateway-resource.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-gateway-resource.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/kubernetes-gateway-simple-http.yml [hunk 3]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-gateway-simple-http.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/kubernetes-gateway-simple-https.yml [hunk 4]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-gateway-simple-https.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/kubernetes-gateway-traefik-lb-svc.yml [hunk 5]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-gateway-traefik-lb-svc.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/kubernetes-whoami-svc.yml [hunk 6]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-whoami-svc.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/networking.x-k8s.io_gatewayclasses.yaml [hunk 7]
```diff
@@ -1,5 +1,3 @@
-
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/networking.x-k8s.io_gatewayclasses.yaml:2]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/networking.x-k8s.io_gateways.yaml [hunk 8]
```diff
@@ -1,5 +1,3 @@
-
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/networking.x-k8s.io_gateways.yaml:2]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/networking.x-k8s.io_httproutes.yaml [hunk 9]
```diff
@@ -1,5 +1,3 @@
-
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/networking.x-k8s.io_httproutes.yaml:2]: Not really in the scope of the PR, would you mind reverting this one?

File: docs/content/reference/dynamic-configuration/kubernetes-gateway-rbac.yml [hunk 10]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (rtribotte) [inline docs/content/reference/dynamic-configuration/kubernetes-gateway-rbac.yml:1]: Not really in the scope of the PR, would you mind reverting this one?

**Review by rtribotte — COMMENTED** [review]
A few things to change ;-)

**Review by rtribotte — APPROVED** [review]
LGTM 👍

**Review by jbdoumenjou — APPROVED** [review]
LGTM :+1:

**Review by dtomcej — APPROVED** [review]
LGTM
:shipit:

**Pull request merged**